backend/test/extensions.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-empty-interface */
+
+import Decimal from 'decimal.js-light'
+
+expect.extend({
+  decimalEqual(received, value) {
+    const pass = new Decimal(value).equals(received.toString())
+    if (pass) {
+      return {
+        message: () => `expected ${received} to not equal ${value}`,
+        pass: true,
+      }
+    } else {
+      return {
+        message: () => `expected ${received} to equal ${value}`,
+        pass: false,
+      }
+    }
+  },
+})
```

Comment:
Is this correct?

If it passes – because it equals – to say then `to not equal` and otherwise `to equal` … 🤔